pkg/rebuild/rebuild/rebuildremote.go

```diff
@@ -138,14 +198,20 @@ var standardBuildTpl = template.Must(
 		"standard build",
 	).Parse(
 		textwrap.Dedent(`
+				#!/usr/bin/env bash
 				set -eux
 				{{- if .UseSyscallMonitor}}
 				touch /workspace/tetragon.jsonl
-				docker run --name=tetragon --detach --pid=host --cgroupns=host --privileged -v=/workspace/tetragon.jsonl:/workspace/tetragon.jsonl -v=/sys/kernel/btf/vmlinux:/var/lib/tetragon/btf quay.io/cilium/tetragon:v1.1.2 /usr/bin/tetragon --export-filename=/workspace/tetragon.jsonl
+				echo '{{.SyscallPolicy}}' > /workspace/tetragon_policy.yaml
+				export TID=$(docker run --name=tetragon --detach --pid=host --cgroupns=host --privileged -v=/workspace/tetragon.jsonl:/workspace/tetragon.jsonl -v=/workspace/tetragon_policy.yaml:/workspace/tetragon_policy.yaml -v=/sys/kernel/btf/vmlinux:/var/lib/tetragon/btf quay.io/cilium/tetragon:v1.1.2 /usr/bin/tetragon --tracing-policy=/workspace/tetragon_policy.yaml --export-filename=/workspace/tetragon.jsonl)
 				{{- end}}
 				cat <<'EOS' | docker buildx build --tag=img -
 				{{.Dockerfile}}
 				EOS
+				{{- if .UseSyscallMonitor}}
+				grep -q "Listening for events..." <(docker logs --follow $TID 2>&1)
```

Comment:
this will fail the script when it fails so you can remove the line below.

per offline discussion, you can also output the logs in the case of failure with ` || docker logs $TID && exit 1`